.eslintrc

```diff
@@ -1,106 +0,0 @@
----
-extends: "airbnb"
-plugins: ["react-hooks"]
-parser: "@babel/eslint-parser"
-globals:
-  page: true
-  browser: true
-  context: true
-  jestPuppeteer: true
-  BASE_URL: true
-rules:
-  camelcase: off # require camel case names
-  prefer-template: off
-  object-shorthand: off
-  quotes: off # ', ", ` no one cares
-  max-len: off # be sensible
-  no-mixed-operators: off # allow 1 + 1 * 3
-  no-confusing-arrow: off # allow (a) => a ? 1 : 2;
-  no-useless-constructor: off
-  no-nested-ternary: off
-  object-curly-spacing: off # allow { a: 1 } and {a: 1}
-  block-spacing: off # same as above for function blocks on a single line
-  dot-notation: off # allows someObject["a"] as well as someObject.a
-  comma-dangle: ["error", "never"]
-  padded-blocks: off
-  no-plusplus: off # we are old enought to use i++
-  arrow-body-style: off
-  arrow-parens: ["error", "always"] # use brackets always, (a) => body // () => body // (a, b) => body
-  no-case-declarations: off # we don't surround case blocks with { } in our switch statements as we use fallthroughs deliberately
-  one-var: off
-  one-var-declaration-per-line: off
-  no-console: [1, { "allow": ["warn", "error"] }]
```

Comment:
Can we bring this rule back please -- it's too easy for a debugging console.log to slip through.

Reply:
(Only really applicable for `./src/` files, so I realise that scripts will need to manually override this rule, which is unfortunate.)

Reply:
We can take advantage of [ESLint config cascading](https://eslint.org/docs/latest/use/configure/configuration-files#cascading-and-hierarchy) to enable this in `src/` only.

Reply:
Done in a9aab1d4 + cf2b5416.